gradoop-flink/src/main/java/org/gradoop/flink/model/api/functions/AggregateFunction.java

```diff
@@ -37,9 +37,34 @@
   PropertyValue aggregate(PropertyValue aggregate, PropertyValue increment);
 
   /**
-   * Sets the property key used to store the aggregate value.
+   * Returns the property key used to store the aggregate value.
    *
    * @return aggregate property key
    */
   String getAggregatePropertyKey();
+
+  /**
+   * Describes the increment of an element that should be added to the aggregate.
+   *
+   * @param element element
```

Comment:
This comment is not very helpful, maybe something like "The element used as the increment."